data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,325 @@
+			`	Along the message comes a small map of the station with the docking ports highlighted.`
+	on complete
+		conversation
+			`Thanks to the map you are easily able to dock with the station. Your airlock opens and you enter an environment like nothing you ever saw before. There are no visible pathways, but instead foggy hallways, some of which even ascend vertically. A crowd of beings gathered in front of your airlock each flash a multitude of colors. A group of five floats in front of the crowd, and the middle one starts pulsating in calm tones.`
+			`	"Welcome, outer-being, to our capital. We are the Incipias."`
+			`	It stops as if awaiting an answer.`
+			choice
+				`	"Greetings, Incipias. I am a human."`
+				`	"I thank you for your hospitality."`
+			`	Your translator begins rapidly flashing. The Incipias seem to respond to this - movement and colors begin to diffuse through the crowd like waves.`
+			`	The 'speaker' continues, "You are the first outer-being that ever communicated with us. Are you the only outer-being out there?"`
+			choice
+				`	"No, there are others too."`
+				`	"The galaxy is full of other beings."`
+			`	Another wave goes through the crowd. "So there are more. But, stranger, how can we help you?"`
+			choice
+				`	(Tell them about the stranded ship and how I want to help them.)`
+					goto stranded
+				`	"I want one of your ships."`
+			`	"We shall allow you to buy one, if that is all that it takes to make you a not-danger. If that is all please excuse us. We need to talk about what this new situation-opportunity means for us. I hope we can out-question you more another time."`
+				goto end
+			label stranded
+			`	Another wave goes through the crowd but this time you notice an overflow in green tones.`
+			`	"So they are still alive. We shall give you access to our ship. Let us prepare the ship-seller. Do note though that they can not jump between system-places, not like your own ship. We hope to question you more on another future-time.`
+			label end
+			`	The five float away but the crowd continues staring at you, probably out of interest.`
+
+mission "Incipias: Help the stranded Part 2"
+	landing
+	passengers 5
+	source
+		planet Turra
+	destination "Pon'tes"
+	to offer
+		has "Incipias: Help the stranded Part 1: done"
+	on offer
+		conversation
+			`Your ship dives into the clouds of the gas giant, an environment on a scale you never experienced before. Giant clouds tower like castles, sometimes opening to giant caverns. Whirlwinds brush through the atmosphere like pillars holding the sky. Even though the winds are brutal your ship glides through them as if they had no effect at all.`
+			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shields must have broke when they ran out of fuel.`
```

Comment:
Shields broke? I thought the Incipias had energy-based shields like other species do.

Reply:
there is not much information on how the shields in ES work, the incipias shields asume that there is some sort of long lasting fuel that would normally never run out, but in this case, it ran out, maybe because of damage on the ship

Reply:
It was mentioned how the Puffin was the only ship that would work for Face to Maw because of its old-type metal shields. But the wiki says [here](https://github.com/endless-sky/endless-sky/wiki/PlayersManual#outfitting-your-ship)
> hull will include an integrated energy shield matrix
However, that is for human ships, and the Incipias might be different. Your choice.

Reply:
@RisingLeaf there is this.

Reply:
again what I mean is not that the shields themself broke but the shield generator, I dont think I made that clear

Reply:
If that is made clear it would help a lot… although that also runs into the question how do you know? Maybe because the ship doesn’t have shields?

Reply:
```suggestion
			`	You activate your scanners looking for the stranded ship. Soon a small blip appears on your screen, and you maneuver your ship to that point. The stranded ship looks even worse than the first time; the storms seem to have damaged some of the hull, and the shield generator must have broke when they ran out of fuel, because their shields are completely down.`
```
Something like this?